stdlib/asm/math/u64.masm

```diff
@@ -6,4 +6,18 @@ export.add_unsafe
     movup.3
     u32addc
     drop
+end
+
+export.mul_unsafe
+    dup.3
+    dup.2
+    u32mul.unsafe
```

Comment:
I guess these are pretty self-explanatory, but it might be nice to have one-liner comments

Reply:
Ah yes! Fixed. (while commenting, the need for #82 became even more apparent).